server/src/main/java/org/opensearch/index/IndexSettings.java

```diff
@@ -578,6 +579,14 @@ public final class IndexSettings {
         Property.InternalIndex
     );
 
+    public static final Setting<String> DEFAULT_SEARCH_PIPELINE = new Setting<>(
```

Comment:
Could we please add field for it here and a getter? (similarly to other settings). As a benefit, we could also check the feature flag and refuse settings update if is it not set (right now the setting is detached from the feature flag).